src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -58,22 +66,6 @@ def auth(self, url: str, name: str, username=None, password=None, token=None, hu
         else:
             multicluster_config = Settings.MULTICLUSTER_CONFIG.copy()
 
-        if 'hub_url' not in multicluster_config:
-            multicluster_config['hub_url'] = hub_url
-
-        if 'config' not in multicluster_config:
-            multicluster_config['config'] = []
-
```

Comment:
why are they removed?

if the config is not there in multicluster_config, this will cause errors below when we try to append the config to dict.

Keep in mind that the user can manually delete the multicluster config from the dashboard settings, and that can break the dashboard.